operators/community-windows-machine-config-operator/5.1.0/manifests/community-windows-machine-config-operator.5.1.0.clusterserviceversion.yaml

```
@@ -0,0 +1,355 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: "false"
+    createdAt: "2022-06-01T09:47:21Z"
+    description: An operator that enables Windows container workloads on OCP
+    olm.skipRange: '>=4.0.0 <5.0.0'
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
+    operators.openshift.io/valid-subscription: '["Red Hat OpenShift support for Windows Containers"]'
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: Red Hat
+  name: community-windows-machine-config-operator-5.1.0
```

Based on past CSVs, the format is slightly different:
```suggestion
  name: community-windows-machine-config-operator.v5.1.0
```

---

Added

---

Is this coming in a commit you haven't pushed yet?

---

Thanks for catching that - I think it reverted back by accident before one of my last pushes